src/lib.rs

```diff
@@ -1,253 +1,253 @@
 use std::collections::BTreeSet;
-use std::sync::atomic::{AtomicUsize, Ordering};
-use std::sync::{Arc, Mutex};
+use std::convert::TryInto as _;
+use std::sync::Mutex;
 use std::time::Instant;
 
-use futures::future;
-use futures::task::SpawnExt;
-
-use tracing_facade::trace_scoped;
-
 fn progress_bar_style(task_count: usize) -> indicatif::ProgressStyle {
-  let task_count_digits = task_count.to_string().len();
-  let count = "{pos:>".to_owned() + &(8 - task_count_digits).to_string() + "}/{len}";
-  let template = "[{elapsed_precise}] {prefix} ".to_owned() + &count + " {bar:40.cyan/blue}: {msg}";
-  indicatif::ProgressStyle::default_bar()
-    .template(&template)
-    .progress_chars("##-")
+    let task_count_digits = task_count.to_string().len();
+    let count = "{pos:>".to_owned() + &(8 - task_count_digits).to_string() + "}/{len}";
```

Comment:
nit: Switch this over to `format!`

```suggestion
    let count = format!("{{pos:>{}}}/{{len}}", 8 - task_count_digits);
```

Reply:
Done.